client/tests/webdriver/pages/home.page.js

```diff
@@ -5,6 +5,14 @@ class Home extends Page {
     get securityBanner() { return browser.element('#topbar .banner') }
     get searchBar() { return browser.element('#searchBarInput') }
     get submitSearch() { return browser.element('#topbar #searchBarSubmit') }
+
+    waitForSecurityBannerValue(value) {
+        this.securityBanner.waitForExist()
+		this.securityBanner.waitForVisible()
```

Comment:
Indentation is not correct in here.

Reply:
The indentation (throughout) is currently a mixture of spaces and tabs, which shows here.